imgui-app/src/main/java/imgui/app/Window.java

```diff
@@ -90,14 +87,29 @@ protected void initWindow(final Configuration config) {
         }
 
         GLFW.glfwMakeContextCurrent(handle);
+
+        GL.createCapabilities();
+
         GLFW.glfwSwapInterval(GLFW.GLFW_TRUE);
-        GLFW.glfwShowWindow(handle);
 
         if (config.isFullScreen()) {
             GLFW.glfwMaximizeWindow(handle);
         }
+        else {
+            GLFW.glfwShowWindow(handle);
+        }
 
-        GL.createCapabilities();
+        GL32.glClearColor(colorBg.getRed(), colorBg.getGreen(), colorBg.getBlue(), colorBg.getAlpha());
```

Comment:
Please, move this logic to a separate method with relatable javadoc about whtat exactly it does and why it's needed.